### ISSUE TITLE
cannot install on rack 3.0

### ISSUE THREAD

**Issue by graf0** [adam12/roda-unpoly#4]
Hi,

I'm trying to use roda-unpoly for roda app running on ruby 3.3.0 and newest roda 3.76 and rack 3.0.8
I cannot install gem - bundler say:

```
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Could not find compatible versions

Because roda-unpoly >= 0.4.0 depends on rack-unpoly >= 0
  and every version of rack-unpoly depends on rack >= 2.0, < 3.0,
  roda-unpoly >= 0.4.0 requires rack >= 2.0, < 3.0.
And because every version of rackup depends on rack >= 3
  and every version of dfmd depends on rackup ~> 2.1,
  roda-unpoly >= 0.4.0 is incompatible with dfmd >= 0.
So, because every version of dfmd depends on roda-unpoly ~> 0.4
  and Gemfile depends on dfmd >= 0,
  version solving has failed.
```

**Comment by adam12** [comment]
Hello,

I'll see what it takes to make rack-unpoly compatible with Rack 3.

**Comment by adam12** [comment]
I released `rack-unpoly` v0.5.0 which should support Rack 3 now.
